modules/skeleton/react/lib/skeleton.tsx

```diff
@@ -13,7 +13,7 @@ export interface SkeletonProps {
 
 const TRANSPARENCY_POSITION = 45;
 const WHITE_SHEEN_WIDTH = 10;
-const DURATION = 5;
+const DURATION = 2;
```

Comment:
Did you meant to change this value?

Reply:
Yes, i ran it by the design team in PT, but Alex brought up a point that it might behave weird on larger screens?

Reply:
I think changing that value should be from a GH issue explaining why it needs to change. Otherwise that context gets lost. There are so many things that exist without any context.

Reply:
that's understandable, I'll revert this change until we have some more context

Reply:
I found this: https://github.com/Workday/canvas-kit/issues/134

I still think changing this value should be a different PR addressing this issue as well as a rationale for what the final number becomes.